redhat-developer/rhdh-operator
# Default configuration for rhdh audit log

## Description
- Added ability to configure mountPath on default-config's pvcs.yaml
- Configured RHDH profile for enabling audit log by default (in the mounted /var/log/redhat-developer-hub/audit directory). NOTE: dynamic-plugins default config is just a placeholder, we may want to have something else here. 


## Which issue(s) does this PR fix or relate to

https://issues.redhat.com/browse/RHIDP-3241
https://issues.redhat.com/browse/RHIDP-4553

## PR acceptance criteria


## How to test changes / Special notes to the reviewer

The simplest way is to:
- Run cluster with Operator (rhdh profile) 
- Create any Backstage CR (examples/bs1.yaml is fine)
- When deployment starts sh on backstage-backend container and check /var/log/redhat-developer-hub/audit directory
- It must exist and contain log file
